src/app/modules/layout/topbar/topbar.component.ts

```diff
@@ -94,6 +99,7 @@ export class TopbarComponent implements OnInit {
 
   readonly hasRebootRequiredReasons = signal(false);
   readonly shownDialog = signal(false);
+  readonly isExperimentalBuild = toSignal(this.apiService.call('system.experimental'));
```

Comment:
If experimental was an attribute in system.info, could we somehow leverage the system.info being done elsewhere for this?